modules/styles.py

```diff
@@ -1,18 +1,9 @@
-# We need this so Python doesn't complain about the unknown StableDiffusionProcessing-typehint at runtime
-from __future__ import annotations

```

Comment:
TBH `from __future__ import annotations` in all modules would probably be a good idea, but that's beyond this PR.